src/main/scala/cromwell/binding/expression/ValueEvaluator.scala

```diff
@@ -13,11 +13,13 @@ import scala.util.{Failure, Success, Try}
 case class ValueEvaluator(override val lookup: String => WdlValue, override val functions: WdlFunctions[WdlValue]) extends Evaluator {
   override type T = WdlValue
 
-  private def replaceInterpolationTag(string: String, tag: String, lookup: ScopedLookupFunction) =
-    string.replace(tag, lookup(tag.substring(2, tag.length - 1)).valueString)
+  private def replaceInterpolationTag(string: String, tag: String, lookup: ScopedLookupFunction) = {
+    val expr = WdlExpression.fromString(tag.substring(2, tag.length - 1))
+    string.replace(tag, expr.evaluate(lookup, functions).get.valueString)
```

Comment:
It's expected because it's wrapped in a Try in the once place it's called... though I should probably have this return a `Try` in the first place.  I'll change.